### PR TITLE
Implemented interactivity in the to-do list

### PR DESCRIPTION
**Pull request by chelobotix**
In this PR, I fulfilled the following tasks:
- [x] Add updateTask method to Task Class.
- [x] Add event listener to the checkbox (change).
- [x] Update items object's value for completed key upon user actions.
- [x] Implement a removeMultiple method to Task Class for the "Clear all completed" button (array method used: filter()).
- [x] Store the updated array of tasks in local storage

### CODE REVIEW

File: src/task.js [hunk 1]
```diff
@@ -21,10 +21,9 @@ export default class Task {
   }
 
   removeMultiple = (array) => {
-    for (let i = array.length - 1; i >= 0; i -= 1) {
```
Comment (GzimAsani) [inline src/task.js:24]: - ### This inline comment is intended for README file
 - [x] Kindly add the required command for running your project `npm start` 
 - [x] Your live link doesn't send to your page live link, either add it or if you don't have a live link just add N/A or explain that you will add it later.

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello @chelobotix                               :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with your javascript on removing all selected tasks +1: 

 ## Required Changes 
- [ ] Check the comments below :arrow_double_down: 


 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by chelobotix** [comment]
All the observations have been implemented.

**Review by ShadyShawkat — APPROVED** [review]
## Hi @chelobotix, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**